### PR TITLE
Mendelics

### PR DESCRIPTION
**Pull request by dschles**
Separating a single organization's assertion from the rest. This allows for easy comparison to the rest of Clinvar, as well as using this singled-out assertion as the curated variant for annotation. The prefered organization can be entered as a parameter when parsing the xml file.

Also, gob doesn't use int8 and float32. They were used to store values in gob (to save space), but gob saves as int/float64 so reverting back to these.

Plus, removed pubmed and mitomap variants that are in Clinvar with status 2+.

### CODE REVIEW

File: clinvar.go [hunk 1]
```diff
@@ -18,9 +18,10 @@ func main() {
 				Usage: "Build consensus variants for annotation.",
 				Flags: []cli.Flag{
 					&cli.StringFlag{Name: "dir", Required: true, Usage: "Clinvar XML folder."},
+					&cli.StringFlag{Name: "org", Required: false, Value: "500035", Usage: "Print specific organization's assertions. (Mendelics = 500035)"},
```
Comment (lmtani) [inline clinvar.go:21]: A flag da linha 20 deve também ser `Required: false` para que o download automático do xml clinvar funcione.

**Review by lmtani — COMMENTED** [review]
I ran it here, and looks fine. I had to make the change described in the following review comment to have the Clinvar xml downloaded automatically. Besides it, LGTM

File: clinvar.go (hunk 1 above)
Comment (dschles) [reply clinvar.go:21]: Changed.

**Review by lmtani — APPROVED** [review]
LGTM

**Pull request merged**